redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1431 @@
+        msg = _(
+            "Mute Role: {role}\nNotification Channel: {channel}\n" "Default Time: {time}"
+        ).format(
+            role=mute_role.mention if mute_role else _("None"),
+            channel=notification_channel.mention if notification_channel else _("None"),
+            time=humanize_timedelta(default_time) if default_time else _("None"),
+        )
+        await ctx.maybe_send_embed(msg)
+
+    @muteset.command(name="notification")
+    @checks.admin_or_permissions(manage_channels=True)
+    async def notification_channel_set(
+        self, ctx: commands.Context, channel: Optional[discord.TextChannel] = None
+    ):
+        """
+        Set the notification channel for automatic unmute issues.
+
+        If no channel is provided this will be cleared and notifications
+        about issues when unmuting users will not be sent anywhere.
+        """
+        if channel is None:
+            await self.config.guild(ctx.guild).notification_channel.clear()
+            await ctx.send(_("Notification channel for unmute issues has been cleard."))
+        else:
+            await self.config.guild(ctx.guild).notification_channel.set(channel.id)
+            await ctx.send(
+                _("I will post unmute issues in {channel}.").format(channel=channel.mention)
+            )
+
+    @muteset.command(name="role")
+    @checks.admin_or_permissions(manage_roles=True)
+    @checks.bot_has_permissions(manage_roles=True)
+    async def mute_role(self, ctx: commands.Context, *, role: discord.Role = None):
+        """Sets the role to be applied when muting a user.
+
+        If no role is setup the bot will attempt to mute a user by setting
+        channel overwrites in all channels to prevent the user from sending messages.
+        """
+        if not role:
+            await self.config.guild(ctx.guild).mute_role.set(None)
+            del self.mute_role_cache[ctx.guild.id]
+            await self.config.guild(ctx.guild).sent_instructions.set(False)
+            # reset this to warn users next time they may have accidentally
+            # removed the mute role
+            await ctx.send(_("Channel overwrites will be used for mutes instead."))
+        else:
+            await self.config.guild(ctx.guild).mute_role.set(role.id)
+            self.mute_role_cache[ctx.guild.id] = role.id
+            await ctx.send(_("Mute role set to {role}").format(role=role.name))
+        if not await self.config.guild(ctx.guild).notification_channel():
+            command_1 = f"`{ctx.clean_prefix}muteset notification`"
+            await ctx.send(
+                _(
+                    "No notification channel has been setup, "
+                    "use {command_1} to be updated when there's an issue in automatic unmutes."
+                ).format(command_1=command_1)
+            )
+
+    @muteset.command(name="makerole")
+    @checks.admin_or_permissions(manage_roles=True)
+    @checks.bot_has_permissions(manage_roles=True)
```

Comment:
What we actually need to be checking for here is the guild permission, not the channel-specific permission.
```suggestion
    @checks.bot_has_guild_permissions(manage_roles=True)
```